### PR TITLE
fix misleading user performance data

### PR DESCRIPTION
**Pull request by wivwiv**

### CODE REVIEW

File: en_US/README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ EMQX Enterprise is the world’s most scalable and reliable MQTT messaging platf
 
 {% emqxce %}
 
-- **Massive Scale**: Scale to 100 million concurrent MQTT connections with a single EMQX 5.0 cluster.
+- **Massive Scale**: Scale to 100 million concurrent MQTT connections with a single EMQX 5.0 cluster. This figure is 10m for version 4.x.
```
Comment (LenaLenaPan) [inline en_US/README.md:19]:
```suggestion
- **Massive Scale**: Each EMQX node supports up to 2 million concurrent MQTT connections, and each EMQX cluster can support up to 10 million concurrent MQTT connections.
```

Comment (wivwiv) [reply en_US/README.md:19]: 这里的文档还是 4.x 版本，想告诉用户 4.x 只有 10M 连接，同时说 5.0 有更大的连接

File: zh_CN/README.md [hunk 2]
```diff
@@ -34,10 +34,11 @@ EMQX 企业版提供一体化的分布式 MQTT 消息服务和强大的 IoT 规
 
 - **开放源码**：基于 Apache 2.0 许可证完全开源，自 2013 年起 200+ 开源版本迭代。
 - **MQTT 5.0**：100% 支持 MQTT 5.0 和 3.x 协议标准，更好的伸缩性、安全性和可靠性。
-- **海量连接**：单节点支持 500 万 MQTT 设备连接，集群可扩展至1亿并发 MQTT 连接。
+- **海量连接**：单节点支持 500 万 MQTT 设备连接，集群可扩展至 1 亿并发 MQTT 连接（4.x 版本为 1000 万）。
```
Comment (LenaLenaPan) [inline zh_CN/README.md:37]:
```suggestion
- **海量连接**：单节点支持 200 万并发 MQTT 连接，单个 EMQX 集群可扩展至支持 1000 万并发 MQTT 连接。
```

**Review by LenaLenaPan — APPROVED** [review]

**Comment by wivwiv** [comment]
Due to the branch name constraint, there is no way to automatically resolve the conflict to complete the commit so I close it.
The same content was pushed in another pr

**Pull request closed without merging**